app/views/errors/unauthorized.html.haml

```diff
@@ -0,0 +1,8 @@
+= content_for :title do
+  You are not authorised to access this page (401)
+
+%article.content-main.callout--calendar-event
+  = render 'application/alerts'
```

Comment:
Does this use the application layout? Because if so, that layout already renders the alerts.